### PR TITLE
LVJAVS-405 Bug fixing

### PR DESCRIPTION
**Pull request by VitalijF**
I add kid-registration validation. Please check

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/validator/ChildValidator.java [hunk 1]
```diff
@@ -45,6 +45,18 @@ public void validate(Object o, Errors errors) {
         }
 
         String comment = kidToValidate.getComment();
-        if (comment.length() > 250){errors.rejectValue("comment", "registration.kid.comment");}
+
+        if (comment.length() > 250) {
```
Comment (vplytan-amplify) [inline src/main/java/ua/softserveinc/tc/validator/ChildValidator.java]: move these magic number to constants and add javadoc to that constants

File: src/main/webapp/WEB-INF/views/manager/edit-booking.jsp [hunk 2]
```diff
@@ -167,6 +165,20 @@
         </div>
     </div>
 </div>
+<div id="kidCommentMessage" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body-inf">
+                <div align="center">
+                    <br>
+                    <h4></h4>
```
Comment (vplytan-amplify) [inline src/main/webapp/WEB-INF/views/manager/edit-booking.jsp]: remove this ?

File: src/main/webapp/resources/js/comment-modal-message.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ * Created by shuk on 16.11.16.
+ */
+function callCommentDialog()  {
+    $('[id^=comment]').click(function() {
+        var comment = $(this).attr('title');
+            $('#kidCommentMessage').find('h4').html(comment);
+            $('#kidCommentMessage').modal('show');
+
+    });
+}
```
Comment (vplytan-amplify) [inline src/main/webapp/resources/js/comment-modal-message.js]: new line

File: src/main/webapp/resources/js/validation-my-booking-ua.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+function validateDate() {
+    var startDate = document.getElementById("from").value.toString().replace(/-/gi,"");
+    var endDate = document.getElementById("to").value.toString().replace(/-/gi,"");
+    if (endDate-startDate < 0) {
+        document.getElementById("errorDate").innerHTML = "Початкова дата не повинна перевищувати кінцеву";
+    } else {
+        document.getElementById("errorDate").innerHTML = "";
+    }
+}
```
Comment (vplytan-amplify) [inline src/main/webapp/resources/js/validation-my-booking-ua.js]: new line

File: src/main/webapp/resources/js/validation-my-booking.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+function validateDate() {
+    var startDate = document.getElementById("from").value.toString().replace(/-/gi,"");
+    var endDate = document.getElementById("to").value.toString().replace(/-/gi,"");
+    if (endDate-startDate < 0) {
+        document.getElementById("errorDate").innerHTML = " 'To' date should be bigger than 'From' date ";
+    } else {
+        document.getElementById("errorDate").innerHTML = "";
+    }
+}
```
Comment (vplytan-amplify) [inline src/main/webapp/resources/js/validation-my-booking.js]: new line

**Comment by hrybs** [comment]
Please, close this pull request.

**Pull request closed without merging**